samples/basic-component/src/components/BasicComponent/BasicComponent.tsx

```diff
@@ -23,7 +23,10 @@ const BasicComponent = (
                 I am a basic component
             </Typography>
             {/* Change the underlying model property value to trigger a re-render */}
-            <Button onClick={() => (model.hidden = !model.hidden)}>
+            <Button
+                data-test="BasicComponent-btn"
```

Comment:
@eric-parsons I added this for testing purposes. Just wanted to make sure you're OK with it.

Reply:
It's not a showstopper, but I'm curious why it's needed?

Reply:
The test for this component was failing. Adding a `wait(1000)` or targeting the element with the data-test worked and I thought targeting the element would be more stable.